packages/rocketchat-lib/server/startup/settings.js

```diff
@@ -628,18 +628,31 @@ RocketChat.settings.addGroup('OAuth', function() {
 });
 
 RocketChat.settings.addGroup('General', function() {
+	this.add('Server_First_Access', true, {
+		type: 'boolean',
+		public: true,
+		blocked: true
+	});
 	this.add('Site_Url', typeof __meteor_runtime_config__ !== 'undefined' && __meteor_runtime_config__ !== null ? __meteor_runtime_config__.ROOT_URL : null, {
 		type: 'string',
 		i18nDescription: 'Site_Url_Description',
 		'public': true
 	});
 	this.add('Site_Name', 'Rocket.Chat', {
 		type: 'string',
-		'public': true
+		'public': true,
+		wizard: {
```

Comment:
@karlprieb can you explain how this new `wizard` property works?

Reply:
I'm using this property to get all setup wizard settings on the setup wizard page.